neomodel/match.py

```diff
@@ -7,13 +7,24 @@
 
 def rel_helper(**rel):
     if rel['direction'] == OUTGOING:
-        stmt = '-[{0}:{1}]->'
+        stmt = '-{0}->'
     elif rel['direction'] == INCOMING:
-        stmt = '<-[{0}:{1}]-'
+        stmt = '<-{0}-'
     else:
-        stmt = '-[{0}:{1}]-'
-    ident = rel['ident'] if 'ident' in rel else ''
-    stmt = stmt.format(ident, rel['relation_type'])
+        stmt = '-{0}-'
```

Comment:
Could this just be simplified to always utilize the bidirectional statement of `'-{0}-'`? I believe the [performance is the same regardless of direction](http://graphaware.com/neo4j/2013/10/11/neo4j-bidirectional-relationships.html). Unless this is to help simplify the logic of the helper to a user. @Hexadite-Max What are your thoughts?